plugins/extractors/bigquery/bigquery.go

```diff
@@ -415,44 +415,49 @@ func (e *Extractor) buildAsset(ctx context.Context, t *bigquery.Table, md *bigqu
 		}
 	}
 
-	table, err := anypb.New(&v1beta2.Table{
-		Columns:       e.buildColumns(ctx, md.Schema, md),
-		PreviewFields: previewFields,
-		PreviewRows:   previewRows,
-		Profile:       tableProfile,
-		Attributes: utils.TryParseMapToProto(map[string]interface{}{
-			"full_qualified_name": tableFQN,
-			"dataset":             t.DatasetID,
-			"project":             t.ProjectID,
-			"type":                string(md.Type),
-			"partition_data":      partitionData,
-			"clustering_fields":   clusteringFields,
-		}),
-		CreateTime: timestamppb.New(md.CreationTime),
-		UpdateTime: timestamppb.New(md.LastModifiedTime),
-	})
-	if err != nil {
-		e.logger.Warn("error creating Any struct", "error", err)
-	}
-
 	asset := &v1beta2.Asset{
 		Urn:         tableURN,
 		Name:        t.TableID,
 		Type:        "table",
 		Description: md.Description,
 		Service:     "bigquery",
-		Data:        table,
 		Labels:      md.Labels,
 	}
 
-	if e.config.BuildViewLineage && (md.Type == bigquery.ViewTable || md.Type == bigquery.MaterializedView) {
+	attributesData := map[string]interface{}{
```

Comment:
Please check if we can add a test case for this, rest all looks good

Reply:
similar discussion [here](https://github.com/goto/meteor/pull/37#discussion_r1325382904), i think we could skip this for now